build_release.sh

```
@@ -104,22 +107,22 @@ CMAKE_ARGS="-DUSE_RERUN=$USE_RERUN -DCMAKE_BUILD_TYPE=$BUILD_TYPE"
 if [ -n "$CUDA_ARCH" ]; then
   CMAKE_ARGS="$CMAKE_ARGS -DCMAKE_CUDA_ARCHITECTURES=$CUDA_ARCH"
 fi
-cmake $CMAKE_ARGS -S $SRC -B $DST
+cmake $CMAKE_ARGS ${EXTRA_CMAKE_ARGS:-} -S $SRC -B $DST
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)